consensus/core/src/stake_aggregator.rs

```diff
@@ -18,12 +19,18 @@ impl CommitteeThreshold for QuorumThreshold {
     fn is_threshold(committee: &Committee, amount: Stake) -> bool {
         committee.reached_quorum(amount)
     }
```

Comment:
nit: add new line in between methods here and in `impl CommitteeThreshold for ValidityThreshold`